packages/platform-browser/test/hydration_spec.ts

```diff
@@ -10,6 +10,7 @@ import {DOCUMENT} from '@angular/common';
 import {HttpClient, provideHttpClient} from '@angular/common/http';
 import {HttpTestingController, provideHttpClientTesting} from '@angular/common/http/testing';
 import {ApplicationRef, Component, Injectable} from '@angular/core';
+import {SSR_CONTENT_INTEGRITY_MARKER} from '@angular/core/src/hydration/utils';
```

Comment:
I think we should avoid deep-linking into the package, we can just import the symbol like this:
```suggestion
import {ApplicationRef, Component, Injectable, ɵSSR_CONTENT_INTEGRITY_MARKER as SSR_CONTENT_INTEGRITY_MARKER} from '@angular/core';
```